cmd/dlt/oidcconfig/cmd.go

```diff
@@ -152,12 +140,26 @@ func buildOidcConfigInput(r *rosa.Runtime) OidcConfigInput {
 		os.Exit(1)
 	}
 	secretArn := oidcConfig.SecretArn()
-	parsedSecretArn, _ := arn.Parse(secretArn)
-	if args.region != parsedSecretArn.Region {
-		r.Reporter.Errorf("Secret region '%s' differs from chosen region '%s', "+
-			"please run the command supplying region parameter.", parsedSecretArn.Region, args.region)
-		os.Exit(1)
+	bucketName := ""
+	if !oidcConfig.Managed() {
+		parsedSecretArn, _ := arn.Parse(secretArn)
+		if args.region != parsedSecretArn.Region {
+			r.Reporter.Errorf("Secret region '%s' differs from chosen region '%s', "+
+				"please run the command supplying region parameter.", parsedSecretArn.Region, args.region)
+			os.Exit(1)
+		}
+		secretResourceName, err := aws.GetResourceIdFromSecretArn(secretArn)
+		if err != nil {
+			r.Reporter.Errorf("There was a problem parsing secret ARN '%s' : %v", secretArn, err)
+			os.Exit(1)
+		}
+		bucketName = strings.TrimPrefix(secretResourceName, prefixForPrivateKeySecret)
+		index := strings.LastIndex(bucketName, "-")
```

Comment:
Can we add an example with a bucket name? so it would be easier to understand the logic of getting the bucket name and the index?

Reply:
Oh sure!